app/models/organisations_user.rb

```diff
@@ -1,19 +1,37 @@
+# frozen_string_literal: true
+
 class OrganisationsUser < ActiveRecord::Base
   module Status
-    PENDING   = 'pending'.freeze
-    APPROVED  = 'approved'.freeze
-    EXPIRED   = 'expired'.freeze
-    DENIED    = 'denied'.freeze
```

Comment:
Any reason for removing the `freeze` ?

Reply:
I have added a magic comment at top `# frozen_string_literal: true` 
this handles the `.freeze` of strings : )